src/Indexer.js

```diff
@@ -109,7 +109,7 @@ export default class Indexer {
       for (const index of Object.keys(this.indexers)) {
         const indexExists = await this.client.indices.exists({ index: index })
 
-        if (!indexExists) {
+        if (!indexExists.body) {
```

Comment:
`indexExists` is an object here, not a boolean, so it's truthy. Use the boolean in `body` instead.